client/src/containers/Topic/Topic/TopicData/TopicData.jsx

```diff
@@ -230,6 +242,18 @@ class TopicData extends React.Component {
     });
   }
 
+  async handleOnShare(row) {
```

Comment:
I think I mostly also want to go the page to see the share result ?

Reply:
open in a new window?

Reply:
Just go to the share page in the same window it will be fine